packages/cli/lib/lib/webpack/webpack-base-config.js

```diff
@@ -269,6 +269,11 @@ module.exports = function (env) {
 							},
 						},
 					],
+					// Don't consider CSS imports dead code even if the
+					// containing package claims to have no side effects.
+					// Remove this when webpack adds a warning or an error for this.
+					// See https://github.com/webpack/webpack/issues/6571
```

Comment:
[Comment taken right from CRA](https://github.com/facebook/create-react-app/blob/65d8eb23f28475716e7e9497215cd574c1bf0b6c/packages/react-scripts/config/webpack.config.js#L498), I felt that having an explanation in the source is useful.